index.js

```diff
@@ -136,6 +138,8 @@ HomeAssistantPlatform.prototype = {
           accessory = new HomeAssistantSwitch(that.log, entity, that, 'scene')
         }else if (entity_type == 'media_player' && entity.attributes && entity.attributes.supported_media_commands){
           accessory = new HomeAssistantMediaPlayer(that.log, entity, that)
+        }else if (entity_type == 'thermostat'){
```

Comment:
Yeah, it's climate now. thermostat is still around and deprecated, so maybe include both?